client/app/bundles/shared/components/Dropdown/Dropdown.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import css from './Dropdown.scss';
+
+import enYML from '../../../../../../config/locales/en.yml';
+
+
+export default class Dropdown extends Component {
+  render() {
+    const options = Object.keys(enYML.en.languages).map((key, index) =>
+      <option value={key} key={index} > {enYML.en.languages[key]} </option>,
+    );
+    return (
+      <div className={css.select_dropdown}>
```

Comment:
Let's make sure this container is `inline-block` in case they are used beside other components in the future!